dhis-2/dhis-services/dhis-service-analytics/src/main/java/org/hisp/dhis/analytics/event/data/AbstractJdbcEventAnalyticsManager.java

```diff
@@ -897,12 +897,12 @@ else if ( header.getValueType() == ValueType.REFERENCE )
     }
 
     /**
-     * Return SQL string bnased on Items a its params
+     * Return SQL string based on Items and its params
      *
      * @param params a {@see EventQueryParams}
      * @param hlp a {@see SqlHelper}
      */
-    protected String getItemsSql( EventQueryParams params, SqlHelper hlp )
+    protected String getItemsAndItemFiltersSql( EventQueryParams params, SqlHelper hlp )
```

Comment:
Maybe we could rename this method to something like "getStatementForDimensionsAndFilters".